nasa-gibs/worldview
# Remove distraction free time component

## Description

Fixes #3430  .

- [x] Remove distraction free time component
- [x] Refactor distraction free alert to integrate localStorage dismissal
- [x] Revise some conditional distraction free component rendering
- [x] Update e2e
 - distraction free changes
 - timeline mobile - revise time pause padding for flakey date change tests

## How To Test

1. Open distraction free mode from the info menu - dismiss alert and also note no time component
2. Refresh page, you should still be in distraction free mode with no alert displayed


## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview
